hw/ip/lc_ctrl/data/lc_ctrl.hjson

```diff
@@ -335,6 +335,24 @@
     { name: "BUS.INTEGRITY",
       desc: "End-to-end bus integrity scheme."
     }
+    { name: "ENCODING.SPARSE",
```

Comment:
To me `ENCODING` doesn't sound right as asset. Would `STATE` be better or is this too generic?